### PR TITLE
Add 1.23.3 to e2e kind tests

### PR DESCRIPTION
**Pull request by naemono**

### CODE REVIEW

File: .ci/pipelines/e2e-tests-kind-k8s-versions.Jenkinsfile [hunk 1]
```diff
@@ -102,7 +102,28 @@ pipeline {
...
                 }
-
+                stage("1.23.3 IPv4") {
+                    agent {
+                        label 'eck'
+                    }
+                    steps {
+                        unstash "source"
+                        script {
+                            runTests(lib, failedTests, "kindest/node:v1.22.0@sha256:0df8215895129c0d3221cda19847d1296c4f29ec93487339149333bd9d899e5a", "0.11.1", "ipv4")
```
Comment (barkbay) [inline .ci/pipelines/e2e-tests-kind-k8s-versions.Jenkinsfile:112]:
```suggestion
                            runTests(lib, failedTests, "kindest/node:v1.23.3@sha256:0df8215895129c0d3221cda19847d1296c4f29ec93487339149333bd9d899e5a", "0.11.1", "ipv4")
```

File: .ci/pipelines/e2e-tests-kind-k8s-versions.Jenkinsfile [hunk 2]
```diff
@@ -102,7 +102,28 @@ pipeline {
...
+                    }
+                }
+                stage("1.23.3 IPv6") {
+                    agent {
+                        label 'eck'
+                    }
+                    steps {
+                        unstash "source"
+                        script {
+                            runTests(lib, failedTests, "kindest/node:v1.22.0@sha256:0df8215895129c0d3221cda19847d1296c4f29ec93487339149333bd9d899e5a", "0.11.1", "ipv6")
```
Comment (barkbay) [inline .ci/pipelines/e2e-tests-kind-k8s-versions.Jenkinsfile:123]:
```suggestion
                            runTests(lib, failedTests, "kindest/node:v1.23.3@sha256:0df8215895129c0d3221cda19847d1296c4f29ec93487339149333bd9d899e5a", "0.11.1", "ipv6")
```

File: .ci/pipelines/e2e-tests-kind-k8s-versions.Jenkinsfile (hunk 1 above)
Comment (naemono) [reply .ci/pipelines/e2e-tests-kind-k8s-versions.Jenkinsfile]: thanks.

**Review by barkbay — APPROVED** [review]
LGTM, I'm just wondering if we want to drop `1.18` ?

**Pull request merged**